src/lib/components/sideTrack/sideTrack.module.scss

```diff
@@ -67,7 +68,8 @@
 
 .sideTrackTitle {
   @include themify() {
-    font-family: themed('brandTertiary');
+    //font-family: themed('brandTertiary');
```

Comment:
I should have removed this comment sorry... Can we remove it now? There will be lots more to follow below.

Reply:
Done